docs/judge/problem_format.md

```diff
@@ -75,7 +75,7 @@ A problem with many possible outputs (e.g. not a single possible answer, with sc
 A `checker` Python script must implement a function that is called by the judge:
 
 ```python
-def check(proc_output, judge_output, judge_input, point_value, submission_source, **kwargs)
+def check(proc_output, judge_output, judge_input, point_value, submission_source, **kwargs):
```

Comment:
Amusingly enough it's always been that way, except we were passing them (and probably still are) in such a way that that format was OK.